pkg/controller/import-controller.go

```diff
@@ -558,9 +558,7 @@ func (r *ImportReconciler) createImportEnvVar(pvc *corev1.PersistentVolumeClaim)
 	return podEnvVar, nil
 }
 
-func (r *ImportReconciler) isInsecureTLS(pvc *corev1.PersistentVolumeClaim) (bool, error) {
-	var configMapName string
-
+func (r *ImportReconciler) isInsecureTLS(pvc *corev1.PersistentVolumeClaim, cdiConfig *cdiv1.CDIConfig) (bool, error) {
```

Comment:
I think the `cdiConfig` parameter should be of type `*CDIConfigSpec` instead of `CDIConfig` that is not the same as the `CDI` type.

Reply:
The CDIConfig resource is a read only copy of what is in the CDI CR